analyses/cnv-comparison/01-cnv-comparison-plotting.R

```diff
@@ -0,0 +1,123 @@
+# Plot and compare detected CNV aberrations given CNVkit and Control-FREEC 
+# output
+#
+# Chante Bethell for CCDL 2019
+#
+# Usage:
+# This script is intended to be run via the command line from the top directory 
+# of the repository as follows:
+#
+# Rscript analyses/cnv-comparison/01-cnv-comparison-plotting.R
+#
+
+#### Install packages ----------------------------------------------------------
+if (!("GenVisR" %in% installed.packages())) {
+  install.packages("BiocManager")
+  BiocManager::install("GenVisR")
+}
+
+if (!("cowplot" %in% installed.packages())) {
+  install.packages("BiocManager")
+  BiocManager::install("cowplot")
```

Comment:
Is there a reason not tin install this from cran?

Reply:
Good question, I cannot pinpoint my initial motivation here so I can change this to install from cran.